### PR TITLE
Introduce cookieName attribute to allow a custom cookie name

### PR DESCRIPTION
**Pull request by dmitry-zaets**
Closes https://github.com/segmentio/consent-manager/issues/125
Adds support of `cookieName` attribute, to customize cookie name

### CODE REVIEW

File: src/consent-manager-builder/preferences.ts [hunk 1]
```diff
@@ -63,8 +64,8 @@ export function savePreferences({
     destinations: destinationPreferences,
     custom: customPreferences
   }
-
-  cookies.set(COOKIE_KEY, value, {
+  debugger
```
Comment (pooyaj) [inline src/consent-manager-builder/preferences.ts:67]: Remove

**Review by pooyaj — COMMENTED** [review]
Sorry for the delay on this. Looks like this one needs a rebase/update. Happy to review/merge when updated

File: src/consent-manager-builder/preferences.ts (hunk 1 above)
Comment (dmitry-zaets) [inline src/consent-manager-builder/preferences.ts:67]:
```suggestion
```

**Comment by dmitry-zaets** [comment]
@pooyaj I've updated the PR and resolved the conflicts

**Comment by pooyaj** [comment]
Awesome! thank you 💥

**Review by pooyaj — APPROVED** [review]

**Pull request merged**